docs/source/tutorials/slowmo_ddp.rst

```diff
@@ -0,0 +1,69 @@
+Efficient Data Parallel Training with SlowMo Distributed Data Parallel
+======================================================================
+
+SlowMo Distributed Data Parallel reduces the communication between different
+nodes while performing data parallel training. It is mainly useful for use on
+clusters with low interconnect speeds between different nodes. When using
+SlowMo, the models on the different nodes are no longer kept in sync after each
+iteration, which leads to the optimization dynamics being affected. The end
+result is close to the results of Distributed Data Parallel, but is not exactly
+the same.
+
+If you have code that is setup to use Distributed Data Parallel, using SlowMo Distributed Data Parallel
+is simply replacing the DDP call with a call to
+``fairscale.experimental.nn.data_parallel.SlowMoDistributedDataParallel``, adding a
+``model.perform_slowmo(optimizer)`` call after ``optimizer.step()``, and moving the
+``model.zero_grad(set_to_none=True)`` to be after ``optimizer.step()``, as follows.
```

Comment:
```suggestion
``fairscale.experimental.nn.data_parallel.SlowMoDistributedDataParallel``, and adding a
```suggestion
``model.perform_slowmo(optimizer)`` call after ``optimizer.step()`` -- preceded by
``model.zero_grad(set_to_none=True)`` in order to reduce peak memory usage.
`````````